projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import subprocess
+
+
+class VDK:
```

Comment:
nit: I would rather use VdkUI or similar, to emphasise it's rather a VDK GUI (web) client than VDK (core,) itself